lint_clean_files.sh

```diff
@@ -10,7 +10,7 @@ mapfile -t FILES < <(
 	cat clean_files.txt \
 		| grep -v -E '^\s*$' \
 		| grep -v -E '^\s*#' \
-		| xargs -n1 -I{} find "{}" -type f
```

Comment:
I don't see where this version is engaging `"max args"` logic?

Reply:
@davidpfarrell do you have a mac on hand to test? odd that it passed the checks..

Reply:
Hey @cornfeedhobo:
Here's a freshly run lint test with current main and newest version of all tooling:

![image](https://user-images.githubusercontent.com/45741407/166987896-a85cc0af-6d46-4dfc-9c8f-7f648f21dd3e.png)

Q: Do you perhaps have an `xargs` alias defined in your local that might contain the `--max-args` option?  Or does xargs honor some kind of `$XARGS_OPT` variable that may have it?

Reply:
> Do you perhaps have an xargs alias defined in your local that might contain the --max-args option? Or does xargs honor some kind of $XARGS_OPT variable that may have it?

No, it's the `-n` flag. Here's the excerpt from linux man page
```plain
-n max-args, --max-args=max-args
              Use at most max-args arguments per command line.  Fewer than max-args arguments will be used if the size (see the -s option) is exceeded, unless the -x  option  is
              given, in which case xargs will exit.
```

Reply:
Doh! Of course `-n` is an alias for `--max-args`. I should have checked that before asking the question.

So looks like `-I {}` will use the whole line in the replacement, so removing the `-n` seems like the fix here.

I would keep the replacement arg as `-I` since mac does not seem to support `--replace`

Reply:
@davidpfarrell Hey no problem at all.

@tbhaxor Would you mind making the suggested changes?

Reply:
@cornfeedhobo I don't have macOS right now. I will test in few days